plugins/inventory/cobbler.py

```diff
@@ -296,8 +306,45 @@ def parse(self, inventory, loader, path, cache=True):
                 self.inventory.add_child(self.group, hostname)
 
             # Add host variables
+            ip_address = None
+            ipv6_address = None
+            for iname, ivalue in interfaces.items():
+                # Set to first interface or management interface if defined or hostname matches dns_name
+                if ivalue['ip_address'] != "":
+                    if ip_address is None:
+                        ip_address = ivalue['ip_address']
+                    elif ivalue['management']:
+                        ip_address = ivalue['ip_address']
+                    elif ivalue['dns_name'] == hostname and ip_address is None:
+                        ip_address = ivalue['ip_address']
+                if ivalue['ipv6_address'] != "":
+                    if ipv6_address is None:
+                        ipv6_address = ivalue['ipv6_address']
+                    elif ivalue['management']:
+                        ipv6_address = ivalue['ipv6_address']
+                    elif ivalue['dns_name'] == hostname and ipv6_address is None:
+                        ipv6_address = ivalue['ipv6_address']
+
+                # Collect all interface name mappings for adding to group vars
+                if self.get_option('want_ip_addresses'):
+                    if ivalue['dns_name'] != "":
+                        if ivalue['ip_address'] != "":
+                            ip_addresses[ivalue['dns_name']] = ivalue['ip_address']
+                        if ivalue['ipv6_address'] != "":
+                            ip_addresses[ivalue['dns_name']] = ivalue['ipv6_address']
+
+            # Add ip_address to host if defined
+            if ip_address is not None:
+                self.inventory.set_variable(hostname, 'cobbler_ipv4_address', ip_address)
+            if ipv6_address is not None:
+                self.inventory.set_variable(hostname, 'cobbler_ipv6_address', ipv6_address)
+
             if self.get_option('want_facts'):
                 try:
                     self.inventory.set_variable(hostname, 'cobbler', host)
                 except ValueError as e:
                     self.display.warning("Could not set host info for %s: %s" % (hostname, to_text(e)))
+
+        if self.get_option('want_ip_addresses'):
+            self.inventory.set_variable(self.group, 'cobbler_ipv4_addresses', ip_addresses)
+            self.inventory.set_variable(self.group, 'cobbler_ipv6_addresses', ipv6_addresses)
```

Comment:
`cobbler_ipv4_addresses` and `cobbler_ipv6_addresses` are neither mentioned in the changelog nor in the documentation.